tests/components/test_smoothing.cpp

```diff
@@ -29,15 +35,15 @@ TEST_CASE("smoothing_Newton_Method")
         scheduler.add_operation_type<operations::tri_mesh::VertexSmoothUsingDifferentiableEnergy>(
             "optimize_vertices",
```

Comment:
Why is there the `spdlog::info("HJELL?");`? That loos a lot like debug code that should be removed.